app/src/main/java/fr/free/nrw/commons/category/CategoryClient.kt

```diff
@@ -8,6 +8,8 @@ import javax.inject.Singleton
 const val CATEGORY_PREFIX = "Category:"
 const val SUB_CATEGORY_CONTINUATION_PREFIX = "sub_category_"
 const val PARENT_CATEGORY_CONTINUATION_PREFIX = "parent_category_"
+const val CATEGORY_UNCATEGORISED = "uncategorised"
+const val CATEGORY_NEEDING_CATEGORIES = "needing categories"
```

Comment:
Are we sure that this will work? The "uncategorized" category is actually "Media needing categories", see https://commons.wikimedia.org/wiki/File:Horse_transport.jpg

Reply:
Ah whoops looked at the wrong section, this should work. :)